core/rawdb/freezer.go

```diff
@@ -39,6 +40,10 @@ var (
 	// errOutOrderInsertion is returned if the user attempts to inject out-of-order
 	// binary blobs into the freezer.
 	errOutOrderInsertion = errors.New("the append operation is out-order")
+
+	// errInvalidDatadir is returned if the ancient directory specified by user
+	// is a symbolic link.
+	errInvalidDatadir = errors.New("symbolic link datadir is not supported")
```

Comment:
Perhaps make it a bit more descriptive name
```suggestion
	errSymlinkDatadir = errors.New("symbolic link datadir is not supported")
```